src/main/java/bio/terra/cli/command/Version.java

```diff
@@ -1,16 +1,30 @@
 package bio.terra.cli.command;
 
-import java.util.concurrent.Callable;
+import bio.terra.cli.command.helperclasses.CommandSetup;
+import bio.terra.cli.command.helperclasses.FormatFlag;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the second-level "terra version" command. */
 @Command(name = "version", description = "Get the installed version.")
-public class Version implements Callable<Integer> {
+public class Version extends CommandSetup {
 
+  @CommandLine.Mixin FormatFlag formatFlag;
+
+  /** Return value is just the version string. */
   @Override
-  public Integer call() {
-    System.out.println(bio.terra.cli.context.utils.Version.getVersion());
+  protected void execute() {
+    String versionReturnValue = bio.terra.cli.context.utils.Version.getVersion();
```

Comment:
nit: `version`'s probably fine, or just inlining it.

Reply:
done.